app/models/episode.rb

```diff
@@ -48,6 +53,8 @@ class Episode < ApplicationRecord
   validates :segment_count, numericality: {only_integer: true, greater_than: 0, less_than_or_equal_to: MAX_SEGMENT_COUNT}, allow_nil: true
   validate :validate_media_ready, if: :strict_validations
 
+  after_initialize :set_default_feeds, if: :new_record?
+  before_validation :set_default_feeds, if: :new_record?
```

Comment:
Any validation or warning if the episode is not in at least one feed?

Reply:
Yeah, I thought about that too.  I kind of want to ask users for feedback / use-cases on that?